scripts/exregional_run_prepstart.sh

```diff
@@ -626,6 +680,85 @@ EOF
    fi
 fi
 fi
+
+#
+#-----------------------------------------------------------------------
+#
+# Process FVCOM Data
+#
+#-----------------------------------------------------------------------
+#
+if [ "${USE_FVCOM}" = "TRUE" ]; then
+
+  set -x
+  latest_fvcom_file="${FVCOM_DIR}/${FVCOM_FILE}"
+  if [ ${HH} -gt 12 ]; then 
+    starttime_fvcom="$(date +%Y%m%d -d "${START_DATE}") 12"
+  else
+    starttime_fvcom="$(date +%Y%m%d -d "${START_DATE}") 00"
+  fi
+  for ii in $(seq 0 3)
+  do
+     jumphour=$((${ii} * 12))
+     fvcomtime=$(date +%Y%j%H -d "${starttime_fvcom}  ${jumphour} hours ago")
+     fvcom_data_fp="${latest_fvcom_file}_${fvcomtime}.nc"
+     if [ -f "${fvcom_data_fp}" ]; then
+       break 
+     fi
+  done
+#  latest_fvcom_file=$(ls -t ${FVCOM_DIR} | head -n1)
+#  fvcom_data_fp="${FVCOM_DIR}/${latest_fvcom_file}"
```

Comment:
Line 709 and line 710 were commented out.  Are they safe to be removed?

Reply:
Yes, it is safe to remove those lines as we have better way to find the FVCOM data.